submissions/f3d0t/task_js_dom/script.js

```diff
@@ -37,44 +37,41 @@ const createMenuLinks = (array, container) => {
 		let li = document.createElement("li");
 		li.textContent = element.name;
 		li.dataset.id = element.id;
-		li.className = "";
+		li.classList = "";
```

Comment:
?

Reply:
I was added empty class because className.toggle() throw an error when there is no class attribute. For now it's deprecated.